enterprise_access/apps/content_metadata/api.py

```diff
@@ -0,0 +1,110 @@
+"""
+Python API for interacting with content metadata.
+TODO: refactor subsidy_access_policy/content_metadata_api.py
```

Comment:
[nit/sanity check] Is this TODO part of this PR, or is it slated for future work?

Reply:
slated for future work